connect_ext_ppr/webapp.py

```diff
@@ -37,6 +39,37 @@
 )
 class ConnectExtensionXvsWebApplication(WebApplicationBase):
 
+    @router.get(
+        '/deployments/{deployment_id}',
+        summary='Deployment details',
+        response_model=DeploymentSchema,
+    )
+    def get_deployment(
+        self,
+        deployment_id: str,
+        client: ConnectClient = Depends(get_installation_client),
+        db: VerboseBaseSession = Depends(get_db),
+        installation: dict = Depends(get_installation),
+    ):
+        dep = (
+            db.query(Deployment)
+            .filter_by(id=deployment_id, account_id=installation['owner']['id'])
+            .first()
+        )
+        if dep is None:
+            raise ObjectNotFound(deployment_id)
+
+        listings = get_all_info(client)
```

Comment:
@rahulmondal already comment [here](https://github.com/cloudblue/connect-extension-xvs/pull/3#discussion_r1253283974) retrieving/listing data for the particular `product_id`/ `hub_id` associated with the Deployment object/s that user is trying to retrieve/list, I totally agree with that. But we have in representation of Deployment a lot of fields that could be changed at any moment, like `vendor_name, vendor_icon, product_name, product_icon, distributor_name, distributor_icon, hub_name`, all that fields we have to request. Another thing that we've talked with Rahul was to have some kind of cache, but Im not sure if this is in the scope of the MVP of this extension.

Reply:
Good question. The only reason I see - is to avoid data duplication. I thought this moment was already discussed. If not - let's return to the discussion.
To make a decision we need to know the use case. How big amount of data are we going to store and how often are we going to change this data? If there will be a lot of deployment but changes will be rare so better not to store the data. In the opposite case - better to store.
But also worth to mention that to catch every change of every field Jonathan wrote above we need more event handling work.

Reply:
Discussed. Decided to leave as is now